addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -1,6 +1,6 @@
 .o_form_view .o_mass_mailing_mailing_form,
 .o_legacy_form_view.o_mass_mailing_mailing_form {
-    input.o_field_char.o_field_widget {
+    input.o_input {
```

Comment:
this is not ideal as it applies to all inputs in that form view. It looks like it's rather some rule missing for the char_emojis widget, in mail.

I would add
```scss
.o_field_char_emojis input {
    padding-right: 40px; // Avoid overlapping subject text on emoji widget
}
```
in mail, in a emojis_char_field.scss file next to the .js and .xml

And I would remove this from here.

What do you think?

Reply:
Hello @aab-odoo :wave: , Changes are done